### PR TITLE
feat(auth-admin): Sync client environments

### PR DESCRIPTION
**Pull request by GunnlaugurG**
# ...

Attach a link to issue if relevant

## What

Functionality to synch data between available environments

## Why

To be able to synch the environments from another environment

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx [hunk 1]
```diff
@@ -57,25 +124,108 @@ const ContentCard: FC<ContentCardProps> = ({
...
+            justifyContent="spaceBetween"
+            alignItems="baseline"
+          >
+            <Text ref={titleRef} marginTop={2} marginBottom={4} variant="h3">
+              {title}
+            </Text>
+            {intent !== 'none' && (
+              <Box>
+                <DropdownMenu
+                  title="Sync"
```
Comment (hedinne) [inline libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx:155]: This should be translated. Also, shouldn't this be
```
title={inSync ? 'Synced' : 'Sync'}
```
or something like that?

File: libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx [hunk 2]
```diff
@@ -57,25 +124,108 @@ const ContentCard: FC<ContentCardProps> = ({
...
+                                      {formatMessage(m.syncSettings)}
+                                    </Text>
+                                  </button>
+                                )}
+                              </Box>
+                            ),
+                          },
+                        ]),
+                  ]}
+                  key="sync-environment"
```
Comment (hedinne) [inline libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx:222]: Why are you giving it an explicit key?

File: libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx (hunk 1 above)
Comment (GunnlaugurG) [reply libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx:155]: you are right, it should be "Synced" or "Out of sync" according to Figma, I will make the changes

File: libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx (hunk 2 above)
Comment (GunnlaugurG) [reply libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx:222]: Thats a good question, the keys should have been on the items and not the DropDowMenu itself

Comment (GunnlaugurG) [reply libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx:222]: I was getting a error message in the browser (Each child in a list should have a unique "key" prop.
Check the render method of `DropdownMenu`. ) thats why I added this key.

**Comment by hedinne** [comment]
@GunnlaugurG, can you commit the changes so I can approve this PR?

**Comment by GunnlaugurG** [comment]
@hedinne done :D

**Review by hedinne — APPROVED** [review]

**Pull request merged**